mcs_benchmark_data/cli/commands/_command.py

```diff
@@ -41,6 +41,11 @@ def __call__(self, args) -> None:
         """
 
     def _make_new_directory(self, *, file_path: Path, need_init: bool):
+        """
+        Make a new directory at the path specified.
+        @param: file_path the path to the file/directory

```

Comment:
This isn't valid epydoc or Sphinx. (Prefer Sphinx.)
Sphinx would be:
```
:param file_path: the path to ...
```

Reply:
My year of javadocs is starting to haunt me...Sphinx makes much more sense to me